app-web/src/utils/helpers.js

```diff
@@ -61,6 +61,20 @@ export const getTextAndLink = (resourceType, resourcesByType) => {
   return textAndPath;
 };
 
+/** return a combine text result or results depens on how much search results we have
+ * @param {num} numOfResults
+ * @returns {string}
+ */
+export const getSearchResultLable = numOfResults => {
+  let resultLabel;
+  if (numOfResults) {
+    resultLabel = `${numOfResults} Result${numOfResults > 1 ? 's' : ''}`;
+  } else {
+    resultLabel = `No Search Result`;
```

Comment:
This else loop relative to a search bug with github search result,  Patrick will fix it in his pr and I will remove this loop afterward.

Reply:
ok.

Reply:
'No Search Results'